src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String[] buysSupplies = readerFromFile(fromFileName);
+        int buy = supplyBuyResult(buysSupplies, BUY);
+        int supply = supplyBuyResult(buysSupplies, SUPPLY);
+
+        try (FileWriter writerToFile = new FileWriter(toFileName)) {
+            writerToFile.append(SUPPLY).append(",")
+                    .append(String.valueOf(supply))
+                    .append(System.lineSeparator())
+                    .append(BUY).append(",")
+                    .append(String.valueOf(buy))
+                    .append(System.lineSeparator())
+                    .append("result,").append(String.valueOf(supply - buy));
+        } catch (IOException e) {
+            throw new RuntimeException(toFileName + " file wasn't found!", e);
+        }
+    }
+
+    private String[] readerFromFile(String file) {
```

Comment:
let's better make method names verbs -> readFromFile etc
reconsider your methods in this class